libraries/AP_RPM/AP_RPM.cpp

```diff
@@ -161,14 +95,97 @@ void AP_RPM::init(void)
     }
 }
 
+/* 
+PARAMETER_CONVERSION
+Remove when: stable release > 4.2 && date > 14-Aug-2023
```

Comment:
discussed on call, better to put year and month when it went in (so August 2021)